wiki/en/Running-a-Server.md

```diff
@@ -179,131 +261,50 @@ Run `sudo systemctl daemon-reload` to register them for first use.
 
-**Genre**: To allow other people to see your Server on one of the Directories built into the Client, select your desired genre Directory. You should see a confirmation message saying whether your Server has registered successfully. If not, and you leave your Server running, it will keep trying to register until a free slot becomes available.
-
-**Custom**: This allows you to specify a custom directory on which to be listed. See the "Options" tab for the Custom Directory address you want to use.
-
-To run your Server _as_ a Directory, you need to set the Custom Directory address as `localhost` or `127.0.0.1` and set the "Genre" to "Custom". [Read this guide](Directories) for further details.
-
-### My Server Info
-
-When running as a Registered Server this displays the Server's name, city and country so that other users can easily identify it in the Directory listing.
-
-### Chat Welcome Message
-
-The text entered here appears to all users when they join the Server (the chat window will open automatically for them). HTML is also supported.
-
-## Options
-
-<figure><img src="{% include img/en-screenshots/server-window-options.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of Jamulus Server window options"></figure>
-
-### Recording Directory
-
-This sets the path to where the Server's recordings will be stored. With this path set, the "Enable Jam Recorder" function in the Server Setup tab will make recording start once the first person connects to the Server, and stops when the last person leaves. Use the "New Recording" button to create a new sub-directory in which the recordings will be stored from then on. Note that Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
-
-**Note**: When your Server is recording, Clients will display a message that recording is on.
-
-### Custom Directory Address
-
-Leave this field empty unless you need to list your Server on a [Custom Directory](#3-directory) or run a Directory.
+* **Linux users** - Launch the "Jamulus Server" shortcut. Or you can open a terminal window (`CTRL+ALT+t` on Debian and related distros), type `jamulus -s` and hit return.
 
-### Server List Filename
+While most common functions in Jamulus can be set using the GUI, others can only be set using options given in a terminal window when the server is launched. Exactly how you do this will depend on your operating system.
 
-Leave this field empty unless you need to run your Server as a [Directory](#3-directory). When in use, this holds the list of registered Servers whilst restarting the Directory. This prevents the server list appearing "empty" until the Servers re-register.
+For example on Windows, to use a specific settings file, right-click on the Jamulus shortcut and choose “Properties” > Target. Add the necessary arguments to Jamulus.exe:
```

Comment:
Our atypical style guide uses quotes where most technical writing uses **bold**. Alright. Up above, on line 278, we use straight quotes. Here we use the "smart quotes". Which do we prefer here? And in this instance, why is Properties smart-quoted, while Target isn't quoted at all? 

Windows 11 may have made it harder to find the Shortcut option. I had to right-click the EXE icon, choose **Show more options**, and choose **Create shortcut**. Kind of an ordeal! Not sure we should assume Jamulus shortcut is easy to find or make anymore.

Reply:
You're right. Please suggest some rules on that to add here: https://jamulus.io/contribute/Style-and-Tone - thanks!

Reply:
But squealy “smart quotes” are just one dimension of quote hell. What a cross-eyed mess, all squigglin around everywhere. "Properties" > "Target" argh! It's **Properties** -> **Target**. Boom. We should **bold** verbatim interface labels like normal people.

Reply:
OK so let's go with bold verbatim interface labels then. One for the style guide too :-)